en/products/vpn/shared.ftl

```diff
@@ -77,8 +77,8 @@ vpn-shared-wireguard-copyright = { -brand-name-wireguard } is a registered trade
 
 vpn-shared-pricing-variable-heading-v2 = Choose a subscription plan that works for you
 
-# Outdated string
-vpn-shared-pricing-variable-heading = Choose a plan that works for you
+vpn-shared-one-subscription-heading = One subscription for all your devices
+vpn-shared-choose-a-plan-sub-heading = Choose a plan that works for you
```

Comment:
Please note I'm using a new string ID here as this line of copy was previously marked as outdated, and the context for where it is shown has now also changed.